04_model_inference.ipynb

```diff
@@ -2138,28 +1012,39 @@
      "text": [
       "              precision    recall  f1-score   support\n",
       "\n",
-      "           0       0.00      0.00      0.00         3\n",
-      "           1       0.00      0.00      0.00         1\n",
-      "           2       0.00      0.00      0.00         3\n",
-      "           3       0.33      0.50      0.40         2\n",
-      "           4       0.00      0.00      0.00         1\n",
-      "           5       0.00      0.00      0.00         0\n",
-      "           6       0.00      0.00      0.00         1\n",
-      "           7       0.00      0.00      0.00         1\n",
-      "           8       0.00      0.00      0.00         2\n",
-      "           9       0.00      0.00      0.00         2\n",
+      "           0       0.00      0.00      0.00       1.0\n",
+      "           9       0.00      0.00      0.00       0.0\n",
       "\n",
-      "    accuracy                           0.06        16\n",
-      "   macro avg       0.03      0.05      0.04        16\n",
-      "weighted avg       0.04      0.06      0.05        16\n",
+      "    accuracy                           0.00       1.0\n",
+      "   macro avg       0.00      0.00      0.00       1.0\n",
+      "weighted avg       0.00      0.00      0.00       1.0\n",
       "\n"
      ]
     }
    ],
    "source": [
     "# Test model on the dataset\n",
     "preds = model.predict(sample_payload)\n",
-    "print(classification_report(y_test, preds))"
+    "cr = classification_report(y_test, preds)\n",
+    "print(cr)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 138,
+   "id": "49821583",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if ACTION:\n",
```

Comment:
Overall looks good, so we should be getting this value from the user when they run the bash script `run-ttm.sh` and/or we also need to update the workflow https://github.com/redhat-et/time-to-merge-tool/blob/dfd2fcf42504cc1d6cd06392ade7bd5dbedbf4a9/.github/workflows/install-from-config.yaml#L35 to specify how we want to get this variable.

Reply:
So I don't think this should update the `run-ttm.sh` as the functionality in this PR is specific to an independent inference step that is not part of the TTM training.  

An upcoming PR, will address the workflow. There will be a new workflow that uses only this notebook to infer TTM for a new PR. I don't think the existing workflows should be updated.

Reply:
Sounds good, @MichaelClifford :+1: